pages/basics/content/fr/install.js

```diff
@@ -0,0 +1,64 @@
+module.exports = {
+  title: 'Installation',
+  subtitle: 'Quelque chose',
+  description: `
+    Ce module est distribué via [npm](https://www.npmjs.com/) qui est empaqueté avec [node](https://nodejs.org) et doit être installé comme une dépendances (~dependencies~) de votre projet :
+
+    ~~~js
+    npm install --save glamorous
+    ~~~
+
+    Il dépend aussi de ~react~ et ~glamor~ donc vous aurez aussi besoin d'eux dans votre projet (si vous ne les possédez pas déjà) :
+
+    ~~~js
+    npm install --save react glamor
+    ~~~
+
+    > REMARQUE : Si vous utilisez React v15.5 ou supérieur, vous aurez également besoin de
+    > ~prop-types~ : ~npm install --save prop-types~
+
+    Vous pouvez ensuite utiliser l'un des formats du module :
+
+    - ~main~: ~dist/glamorous.cjs.js~ - s'exporte en tant que module CommonJS
+    - ~global~: ~dist/glamorous.umd.js~ et ~dist/glamorous.umd.min.js~ - s'exporte en tant que module umd qui est consommable dans plusieurs environnements, le plus significatif de manière globale.
+    - ~jsnext:main~ et le module : ~dist/glamorous.es.js~ - s'exporte en utilisant la spécification des modules ES, vous aurez besoin de configurer webpack pour faire usage de ce fichier,  faites-le en utilisant la propriété resolve.mainFields.
```

Comment:
Ah! Very interesting! I guess that's why we call this "localization" rather than simply "translation" ;-) We can leave it as-is! Thanks!